docs/usage.md

```diff
@@ -159,7 +146,7 @@ You can also supply a run name to resume a specific run: `-resume [run-name]`. U
 
 Specify the path to a specific config file (this is a core Nextflow command). See the [nf-core website documentation](https://nf-co.re/usage/configuration) for more information.
 
-## Custom configuration
+## Custom configuration - unsure how much of this is still relevant for picard profiler? Does all of this still apply?
```

Comment:
It does apply, but I don't think we need to alter the default setup in this case.

```suggestion
## Custom configuration
```